### ISSUE TITLE
Package is still unavailable for install on conda

### ISSUE THREAD

**Issue by rfarouni** [bioconda/bioconda-recipes#6132]
@bioconda/core  Does anyone know why it is taking so long to have the updated package install through conda? I still get v0.5.0, although a couple of weeks ago, I updated to 0.5.1 and 4 days ago I updated to 0.5.2

```
conda install haystack_bio

The following NEW packages will be INSTALLED:

    haystack_bio: v0.5.0-0               bioconda   
```

https://github.com/bioconda/bioconda-recipes/blob/86572a55e3227485ada7c2a3a4cbe03677433cb6/recipes/haystack_bio/meta.yaml#L3

**Comment by daler** [comment]
@rfarouni looks like `skip: True` is set unconditionally: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/haystack_bio/meta.yaml#L20. So it's always skipped no matter what.

If it needs to be skipped for osx, use `skip: True  # [osx]`, or otherwise you can just delete that line to get the package built.

**Comment by daler** [comment]
Closed by #6114
